torch/_torch_docs.py

```diff
@@ -4530,6 +4530,11 @@ def merge_dicts(*dicts):
 
 Returns the median value of all elements in the :attr:`input` tensor.
 
+.. note::
+    For even-sized inputs, the smaller of the two middle elements in the sorted
+    order is returned. If the mean of the two middle elements is required, use
```

Comment:
If the above suggestion is adopted, this would be something like, "To compute the mean of both medians in the input, use...".